GVFS/GVFS.Common/NuGetUpgrader/ReleaseManifest.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+
+namespace GVFS.Common
+{
+    /// <summary>
+    /// Details on the upgrade included in this package, including information
+    /// on what packages are included and how to install them.
+    /// </summary>
+    public abstract class ReleaseManifest

```

Comment:
There is not a difference between upgrades and releases in the manifest... All of the contents of a manifest seem to be related to a particular release and how to install a particular release, rather than any logic for an "upgrade" (e.g. it doesn't check previous versions, etc...).

Reply:
What do you think of `InstallManifest`?